src/puppetlabs/puppetdb/config.clj

```diff
@@ -71,7 +71,8 @@
      :stats (pls/defaulted-maybe String "true")
      :log-statements (pls/defaulted-maybe String "true")
      :statements-cache-size (pls/defaulted-maybe s/Int 0)
-     :connection-timeout (pls/defaulted-maybe s/Int 3000)}))
+     :connection-timeout (pls/defaulted-maybe s/Int 3000)
+     :facts-blacklist String}))
```

Comment:
We should allow the user to provide either strings of comma-separated fact names or an array of fact names; the latter can come from a .conf (HOCON) configuration file.